core/src/test/scala/unit/kafka/controller/ControllerIntegrationTest.scala

```diff
@@ -556,7 +557,7 @@ class ControllerIntegrationTest extends ZooKeeperTestHarness {
     testControllerMove(() => {
       val adminZkClient = new AdminZkClient(zkClient)
       adminZkClient.createTopicWithAssignment(tp.topic, config = new Properties(), assignment)
-    })
+    }, classOf[ControllerIntegrationTest].toString + "#testControllerMoveOnTopicCreation")
```

Comment:
`s"${classOf[ControllerIntegrationTest]}#testControllerMoveOnTopicCreation"`, and if not then there should be no need for the `toString`

Reply:
Oh my, your approach is much simpler and more clear. Okay, I will take this approach.